docs/topics/using-openrewrite-recipes.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following module:
+//
+// * docs/topics/cli-run
+
+[id=using-openrewrite-recipes_{context}]
+= Using OpenRewrite recipes
+
+[IMPORTANT]
+====
+OpenRewrite recipe support is provided as Technology Preview only. Technology Preview features are not supported with Red Hat production service level agreements (SLAs), might not be functionally complete, and Red Hat does not recommend to use them for production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
+
+See link:{KBArticleTechnologyPreview}[Technology Preview features support scope] on the Red&nbsp;Hat Customer Portal for information about the support scope for Technology Preview features.
+====
+
+You can refactor the source code of Java applications by using link:https://docs.openrewrite.org/[OpenRewrite] recipes with the {ProductShortName} CLI.
+
+The OpenRewrite recipe `JavaxtoJakarta` renames imported `javax` packages to their `jakarta` equivalents.
+
+.Procedure
+
+. Specify the OpenRewrite recipe when you run `mta-cli`:
++
+[source,terminal]
+----
+$ ./mta-cli --openrewrite "-DactiveRecipes=<recipe_name>"  \
+    --input </path/to/source/project> --goal dryRun
+----
+
+* `--openrewrite`: Flag for running an OpenRewrite recipe, instead of an {ProductShortName} analysis.
+
+* `"-DactiveRecipes=<recipe name>"` : The OpenRewrite recipe to apply to the input project. `org.jboss.windup.JavaxtoJakarta` is the default shipped recipe.
```

Comment:
Remove space before colon.
Change to this:
 * `"-DactiveRecipes=<recipe name>"`: Specify the OpenRewrite recipe, for example, `JavaxtoJakarta`. 
 
Check whether recipe name is capitalized or all lower case in the actual command.

Reply:
@m-brophy, should the user type `org.jboss.windup.JavaxtoJakarta` for the recipe name or `org.jboss.windup.javaxtojakarta` ?